fee_allocator/accounting/recon.py

```diff
@@ -10,77 +10,69 @@
 from fee_allocator.accounting import PROJECT_ROOT
 
 
-def recon_and_validate(
-    fees: dict,
```

Comment:
We were passing in a static aurabal share here, and then propigating it to a data file...